src/orders/components/OrderRefundDialog/OrderRefundDialog.test.tsx

```diff
@@ -74,6 +78,25 @@ describe("OrderRefundDialog", () => {
     fireEvent.click(confirmButton);
 
     // Assert
-    expect(props.onConfirm).toHaveBeenCalled();
+    expect(props.onStandardRefund).toHaveBeenCalled();
+  });
+  it("calls onManualRefund when the confirm button is clicked", async () => {
```

Comment:
could you use more descriptive test descriptions that show the intention? eg. "it makes a manual refund when the user confirms it"